src/ApiServices/ApiServices.js

```diff
@@ -26,4 +28,38 @@ const logInHandler = async ({ email, password }) => {
   }
 };
 
-export { signUpHandler, logInHandler };
+// Notes
+
+const createNoteHandler = async (data) => {
+  try {
+    const res = await axios.post(
+      "/api/notes",
+      {
+        note: { ...data },
```

Comment:
i guess here spread operator is not needed.

Reply:
Okay removed